lib/assessment.rb

```diff
@@ -5,7 +5,7 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    max { |a, b| a.length <=> b.length }
```

Comment:
Oh! We need to append the passed in argument with a method. New ruby syntax goodness to learn.